### ISSUE TITLE
Evaporate.create() not working with Firefox 61

### ISSUE THREAD

**Issue by fkjaekel** [TTLabs/EvaporateJS#414]
Hi,

On Firefox 61, Evaporate.create() is not getting called. Debugging the code, I can see that it only step over the line, the error callback is not called. The same code works fine with Chrome 67. Any ideias of what is happening? 

I'm using Evaporate 2.1.4.

Thanks

**Comment by fkjaekel** [comment]
[evaporate.zip](https://github.com/TTLabs/EvaporateJS/files/2253382/evaporate.zip)

**Comment by jakubzitny** [comment]
Works for me.

**Comment by fkjaekel** [comment]
After some further digging I found the issue: 
`xAmzHeadersAtInitiate : {'x-amz-meta-lastmodified': mp.files[i].lastModifiedDate.getTime().toString()},`

I had this line in the property list of Evaporate.add(). 

lastModifiedDate was removed on Firefox 61: https://www.fxsitecompat.com/en-CA/docs/2018/file-lastmodifieddate-has-been-removed/

Sorry for the noise, but since there was no output in Firefox console I got a bit lost. It should show uncaught Promise exceptions like Chrome do.
